CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 * The `DistanceFormatter.attributedString(for:)` method is now implemented. It returns an attributed string representation of the distance in which the `NSAttributedStringKey.quantity` attribute is applied to the numeric quantity. ([#1176](https://github.com/mapbox/mapbox-navigation-ios/pull/1176))
 * Fixed an issue in which turn lanes were displayed in the wrong order when the system language was set to Hebrew. ([#1175](https://github.com/mapbox/mapbox-navigation-ios/pull/1175))
 
+#### MapboxCoreNavigation
+* `RouteController`'s `MBRouteControllerProgressDidChange` notification now exposes the raw location in it's update, accessible by `MBRouteControllerRawLocationKey`
```

Comment:
I think we should tell more about the change here. `RouteControllerNotificationUserInfoKey.locationKey` is now the snapped location (before it was raw, raw moved to MBRouteControllerRawLocationKey).

Reply:
Replied [here.](https://github.com/mapbox/mapbox-navigation-ios/pull/1188#issuecomment-370519058)

Reply:
I still think we should note this change. `RouteControllerNotificationUserInfoKey.locationKey ` was the unsnapped (essentially,`rawLocation`) location before.  Now it is the snapped location.